### ISSUE TITLE
RandomX support for Power9

### ISSUE THREAD

**Issue by Cryptonord** [nioroso-x3/xmr-stak-power#26]
A JIT similar to ARM JIT of RandomX has to be written to mine efficiently in PowerPC

**Comment by sk4tto** [comment]
Hi Nioroso,

now RandomX its mandatory.
do you think you will ever update this JIT? it would be really nice to be able to use the power architecture again ...

**Comment by spuhpointer** [comment]
Yeah my S822 with 2x CPU, each of 10 cores would be to happy to do the RandomX.
